### ISSUE TITLE
Success Criterion 4.1.1 is removed in WCAG 2.2

### ISSUE THREAD

**Issue by Jym77** [auto-wcag/auto-wcag#1980]
Following [this WCAG discussion](https://github.com/w3c/wcag/issues/2525), decision has been taken to **remove 4.1.1** from WCAG 2.2, making it effectively not backward-compatible with WCAG 2.1.

This means that our rules that map to 4.1.1 ([Attribute is not duplicated](https://act-rules.github.io/rules/e6952f) and [Id attribute value is unique](https://act-rules.github.io/rules/3ea0c8) are going to map to a non-existing SC (although they might still be relevant for people using 2.0, e.g. Section 508…)

How can we handle that? Should we remove the requirement mapping from the rules? Deprecate the rules altogether? Find a way to say "this maps to a 2.0/2.1 SC but not to a 2.2 one"?

(note that we have a similar but much less strong problem with 2.4.7 changing level)

**Comment by Jym77** [comment]
See also https://github.com/w3c/wcag/issues/2820

Also, [Attribute is not duplicated](https://www.w3.org/WAI/standards-guidelines/act/rules/e6952f/proposed/) is **not** a 4.1.1 failure (in the light of the WCAG discussion), and never was. We should remove the mapping altogether.

**Comment by carlosapaduarte** [comment]
From the [CG call](https://www.w3.org/2023/01/26-act-r-minutes.html):

- Suggest to the TF to deprecate both rules

**Comment by dd8** [comment]
Is it worth maintaining a mapping of how 4.1.1 failures map to other SC? I'm happy to do this if there's any interest.

There have been 2 conflicting interpretations in the WG of the "nested according to specification" requirements:

- content model nesting (e.g. TD must appear inside a TR) which was the interpretation understood by SteveF and was tested by the TPGi parsing bookmarklet
- syntactical nesting which was the original intent and makes sense for XML/XHTML but doesn't make much sense for the HTML5 parsing algorithm (which handles both content model and syntax together when parsing)

Someone on the WG put this together, but it's incomplete:
https://docs.google.com/document/d/1MJ6FxO7ujQ4X9BQtAnDDoWyvpAKU44MR4h-bob9SG7M/edit

There are other issues like wrapping a `<table>` with a `<button>` is not allowed in the content model, and makes the table presentational by removing the table semantics (related to https://www.w3.org/WAI/standards-guidelines/act/rules/307n5z/proposed/)
